packages/kununu-utils/kununu-logger/fingers-crossed/index.js

```diff
@@ -72,29 +77,37 @@ module.exports = class FingersCrossed extends TransportStream {
   hasMinimumLogLevel = level => this.getLogLevel(level) <= this.getLogLevel(this.level);
 
   /**
-   * Push log to state
+   * Save log on state
    *
    * @param {String} traceId
-   * @param {Object} log
+   * @param {Object} raw
    */
-  statePush = (traceId, log) => this.state.push({trace_id: traceId, formatted_log: log});
+  saveOnState = (traceId, raw) => {
+    let log = [];
+
+    try {
+      log = this.cache.get(traceId, true);
+    } catch (err) {
+      this.cache.set(traceId, []);
```

Comment:
I think it's unnecessary to set the cache if an error occurs because, after the try/ catch block, you set the cache again.

Reply:
Makes sense, just changed it to have a default value: `let log = this.cache.get(traceId) || [];`